src/boolean3.cpp

```diff
@@ -512,92 +506,30 @@ Boolean3::Boolean3(const Manifold::Impl &inP, const Manifold::Impl &inQ,
     return;
   }
 
-  // Level 3
-  // Find edge-triangle overlaps (broad phase)
-  p1q2_ = inQ_.EdgeCollisions(inP_);
-  p2q1_ = inP_.EdgeCollisions(inQ_, true);  // inverted
-
-  p1q2_.Sort();
-  PRINT("p1q2 size = " << p1q2_.size());
-
-  p2q1_.Sort();
-  PRINT("p2q1 size = " << p2q1_.size());
-
-  // Level 2
-  // Find vertices that overlap faces in XY-projection
-  SparseIndices p0q2 = inQ.VertexCollisionsZ(inP.vertPos_);
-  p0q2.Sort();
-  PRINT("p0q2 size = " << p0q2.size());
-
-  SparseIndices p2q0 = inP.VertexCollisionsZ(inQ.vertPos_, true);  // inverted
-  p2q0.Sort();
-  PRINT("p2q0 size = " << p2q0.size());
-
-  if (3 * p1q2_.size() + 3 * p2q1_.size() > INT_MAX_SZ) {
-    valid = false;
-    return;
-  }
-
-  // Find involved edge pairs from Level 3
-  SparseIndices p1q1 = Filter11(inP_, inQ_, p1q2_, p2q1_);
-  PRINT("p1q1 size = " << p1q1.size());
-
 #ifdef MANIFOLD_DEBUG
   broad.Stop();
   Timer intersections;
   intersections.Start();
 #endif
 
-  // Level 2
-  // Build up XY-projection intersection of two edges, including the z-value for
-  // each edge, keeping only those whose intersection exists.
-  Vec<int> s11;
-  Vec<vec4> xyzz11;
-  std::tie(s11, xyzz11) = Shadow11(p1q1, inP, inQ, expandP_);
```

Comment:
I love seeing all this removed! That broad phase timer above is probably useless now and can be removed as well.